build.sbt

```diff
@@ -19,21 +19,23 @@ lazy val commonDependencies = Seq(
   Dependencies.Libraries.kinesisClient,
   Dependencies.Libraries.kinesisConnector,
   Dependencies.Libraries.validator,
+  Dependencies.Libraries.nsqClient,  
   // Scala
   Dependencies.Libraries.scopt,
   Dependencies.Libraries.scalaz7,
   Dependencies.Libraries.scalazC7,
   Dependencies.Libraries.snowplowTracker,
   Dependencies.Libraries.analyticsSDK,
   Dependencies.Libraries.awsSigner,
+  Dependencies.Libraries.pureconfig,
   // Scala (test only)
   Dependencies.Libraries.specs2
 )
 
 lazy val buildSettings = Seq(
   organization  := "com.snowplowanalytics",
   name          := "snowplow-elasticsearch-loader",
-  version       := "0.9.0",
+  version       := "0.10.0-rc2",
   description   := "Load the contents of a Kinesis stream to Elasticsearch",
```

Comment:
could you change that to "Load the contents of a Kinesis stream or NSQ topic to Elasticsearch"